packages/perseus-core/src/types.ts

```diff
@@ -1,6 +1,7 @@
 // Types that can be shared between Perseus packages
 // ideally without causing circular dependencies
 
+// TODO: this should be typed
```

Comment:
Should be... but probably extremely hard because each widget kicks back its own, custom-defined state (in some cases the state isn't even an object)!. 

If I remember, this outer state from the Renderer is basically a dictionary of widget ids to their state... so I guess we could start typing it and have:

```js
type State = Record<string, any>; // string being the widget id. 

```